deblur/test/test_workflow.py

```diff
@@ -576,63 +568,60 @@ def test_build_index_sortmerna(self):
         files_to_remove_act = [basename(f) for f in files_to_remove]
         self.assertListEqual(files_to_remove_exp, files_to_remove_act)
 
-    def test_launch_workflow(self):
-        """Test launching complete workflow using simulated sequences 1.
+    def run_workflow_try(self, simfilename, origfilename, ref_fp, ref_db_fp):
+        """Test launching the complete workflow using simulated sequences
+        and compare to original ground truth.
+
+        input:
```

Comment:
This is not following the numpy standards.

Change `input:` to:
```
Parameters
---------------
```

and `string` to `str`